tests/unit/s2n_client_pq_kem_extension_test.c

```diff
@@ -22,6 +22,8 @@ int main(int argc, char **argv)
 {
     BEGIN_TEST();
 
+#if !defined(S2N_NO_PQ)
```

Comment:
We might want to turn this into an "s2n_is_pq_enabled()" check like we do for other flags so we don't have to use the defines everywhere

Reply:
This certainly sounds like a better way. I started to implement this, but there are 34 places where this defined is used. So I created this issue instead: #1952